.github/workflows/main.yml

```diff
@@ -76,11 +72,7 @@ jobs:
         uses: ./.github/actions/setup
         with:
           cargo-cache-key: cargo-semver
-
-      - name: Install cargo-audit
-        uses: taiki-e/install-action@v2
-        with:
-          tool: cargo-semver-checks
+          cargo-semver-checks: true
```

Comment:
Doesn't need to be done in this PR, but since the semver job is run on publish, we can probably remove the job here. It tends to create a lot of false negatives